2022-04-18 18:05:59,033 INFO    MainThread:121049 [wandb_setup.py:_flush():71] setting env: {}
2022-04-18 18:05:59,035 INFO    MainThread:121049 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-18 18:05:59,036 INFO    MainThread:121049 [wandb_init.py:_log_setup():371] Logging user logs to /scratch/users/fsofian19/COMP491_model/models/ast_custom/egs/custom/wandb/run-20220418_180558-22ygka4r/logs/debug.log
2022-04-18 18:05:59,037 INFO    MainThread:121049 [wandb_init.py:_log_setup():372] Logging internal logs to /scratch/users/fsofian19/COMP491_model/models/ast_custom/egs/custom/wandb/run-20220418_180558-22ygka4r/logs/debug-internal.log
2022-04-18 18:05:59,037 INFO    MainThread:121049 [wandb_init.py:init():404] calling init triggers
2022-04-18 18:05:59,038 INFO    MainThread:121049 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-04-18 18:05:59,038 INFO    MainThread:121049 [wandb_init.py:init():460] starting backend
2022-04-18 18:05:59,039 INFO    MainThread:121049 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-18 18:05:59,079 INFO    MainThread:121049 [backend.py:ensure_launched():216] starting backend process...
2022-04-18 18:05:59,098 INFO    MainThread:121049 [backend.py:ensure_launched():221] started backend process with pid: 121111
2022-04-18 18:05:59,108 INFO    MainThread:121049 [wandb_init.py:init():469] backend started and connected
2022-04-18 18:05:59,117 INFO    MainThread:121049 [wandb_init.py:init():533] updated telemetry
2022-04-18 18:05:59,205 INFO    MainThread:121049 [wandb_init.py:init():563] communicating current version
2022-04-18 18:06:04,207 INFO    MainThread:121049 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-18 18:06:07,182 INFO    MainThread:121049 [wandb_init.py:init():606] starting run threads in backend
2022-04-18 18:06:12,208 INFO    MainThread:121049 [wandb_run.py:_console_start():1810] atexit reg
2022-04-18 18:06:12,211 INFO    MainThread:121049 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-04-18 18:06:12,213 INFO    MainThread:121049 [wandb_run.py:_redirect():1689] Redirecting console.
2022-04-18 18:06:12,218 INFO    MainThread:121049 [wandb_run.py:_redirect():1745] Redirects installed.
2022-04-18 18:06:12,218 INFO    MainThread:121049 [wandb_init.py:init():633] run started, returning control to user process
2022-04-18 18:12:09,732 INFO    MainThread:121049 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-04-18 18:12:09,737 INFO    MainThread:121049 [wandb_run.py:_restore():1752] restore
2022-04-18 18:12:12,026 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 18:12:12,254 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 18:12:12,510 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 18:12:12,624 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:12:12,727 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:12:12,832 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:12:12,938 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:12:13,042 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:12:13,146 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:12:13,249 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:12:13,353 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:12:13,459 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:12:13,563 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:12:14,159 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:12:14,724 INFO    MainThread:121049 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}
local_info {
}

2022-04-18 18:12:18,027 INFO    MainThread:121049 [wandb_run.py:_append_files():2180] logging synced files
